src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -3603,7 +3604,7 @@ private Task SNIWritePacket(SNIHandle handle, SNIPacket packet, out UInt32 sniEr
                 {
                     SqlClientEventSource.Log.TryTraceEvent("<sc.TdsParser.WritePacket|Info> write async returned error code {0}", (int)sniError);
                     AddError(_parser.ProcessSNIError(this));
-                    ThrowExceptionAndWarning(callerHasConnectionLock);
+                    ThrowExceptionAndWarning(callerHasConnectionLock, false);
```

Comment:
@cheenamalhotra Did we purposely call `ThrowExceptionAndWarning` with the `asyncClose` parameter hard coded to `false` here? In the same line in netcore, we passed the local value of `asyncClose` to the same parameter. ([Line 3573 in the netcore TdsParserStateObject.cs file above in this PR](https://github.com/dotnet/SqlClient/pull/1766/files#diff-e15040521566aad8d026726b5b8269aeeecdfb7d8e92e3f85c52f37be3dc752fR3573).)

This seems like a mistake and I've changed this line to match netcore in PR #2073.